tests/data/comments7.py

```diff
@@ -183,7 +183,7 @@ def func():
         0.0789,
         0.0123,
         0.0789,
-        a[-1],  # type: ignore
+        a[-1],  #  type: ignore
```

Comment:
Looks like it turns "#<nbsp>type" into "# <nbsp>type". Let's just leave it as is instead of inserting an additional space.

Reply:
I replaced the test data from ``# type: ignore`` to ``#[NBSP]type: ignore`` so now it becomes ``# [NBSP]type: ignore``. That's what we're trying to avoid in other comments, but we have to ignore type comment (or the refactor could be huge to take "apparently invalid type ignore comment turning into valid one" into account).

Reply:
I actually changed my mind on this, I think it's good to add the space here to alert the user that this is an invalid type comment masquerading as a correct one. So let's just leave this as is.

Reply:
I think it's a test case we better add or this edge case with ``type: ignore`` comment would not be tested/apparent.